manifests/katello.pp

```diff
@@ -31,25 +38,30 @@
     mode   => '0755',
   } ->
   # Placing the CA in the pub dir for trusting by a user in their browser
-  file { "${katello_www_pub_dir}/${certs::server_ca_name}.crt":
+  file { "${katello_www_pub_dir}/${certs::katello::server_ca_name}.crt":
     ensure  => file,
-    source  => "${certs::pki_dir}/certs/${certs::server_ca_name}.crt",
+    source  => "${certs::pki_dir}/certs/${certs::katello::server_ca_name}.crt",
     owner   => 'root',
     group   => 'root',
     mode    => '0644',
-    require => File["${certs::pki_dir}/certs/${certs::server_ca_name}.crt"],
+    require => File["${certs::pki_dir}/certs/${certs::katello::server_ca_name}.crt"],
```

Comment:
And here https://github.com/Katello/puppet-certs/blob/master/manifests/init.pp#L130